backend/Controllers/MetadataTypeController.cs

```diff
@@ -141,5 +141,33 @@ public async Task<IActionResult> PostAsync([FromBody] Tag newTag, string name)
 			await _unitOfWork.CompleteAsync();
 			return Ok(res);
 		}
+
+		/// <summary>
+		/// Get list of metadatatype names
+		/// </summary>
+		/// <returns>
+		/// The return type of this is enum list of strings
+		/// </returns>
+		[AllowAnonymous]
+		[HttpGet("by/name")]
+		public async Task<IActionResult> GetByNameAsync()
+		{
+			var resources = await _metadataTypeService.ListNamesAsync();
+			return Ok(resources);
+		}
+
+		/// <summary>
+		/// Get list of metadatatype tags
+		/// </summary>
+		/// <returns>
+		/// The return type of this is enum list of lists of MetadataTypeMapping
+		/// </returns>
+		[AllowAnonymous]
+		[HttpGet("by/tags")]
+		public async Task<IActionResult> GetByTagAsync()
+		{
+			var resources = await _metadataTypeService.ListTagsAsync();
```

Comment:
This isn't mapped properly. See other uses of the automapper.

The output looks like this at the moment:

```
[
  [
    {
      "tagName": "Public activity",
      "tag": null,
      "metadataTypeName": "Car History",
      "type": null
    },
    {
      "tagName": "Traffic",
      "tag": null,
      "metadataTypeName": "Car History",
      "type": null
    }
  ],
  [
    {
      "tagName": "Public activity",
      "tag": null,
      "metadataTypeName": "Cycle History",
      "type": null
    }
  ]
]
```

`tag` and `type` should not be in the response if their foreign key(`tagName` and `metadataTypeName`) are. 

Also, what is being returned isn't tags, its tag mappings

Reply:
This should be resolved at this point.